### PR TITLE
Add notification about successful registration.

### PR DESCRIPTION
**Pull request by doskachok**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -10,9 +9,7 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
```
Comment (AwesomeFlax) [inline src/index.tsx:13]: What does that change and can we avoid it?

Comment (doskachok) [reply src/index.tsx:13]: "[Remove StrickMode it caused the application to render multiple times.](https://github.com/doskachok/magellan/pull/76/commits/225bb7e483909438395165a2b1dbbad5fa5d8b41)"
In this case it executed 2/3 toasts about successful registration.

Comment (AwesomeFlax) [reply src/index.tsx:13]: Well, in all other cases it seemed to work properly, what’s so special about hit this push notification?

Comment (doskachok) [reply src/index.tsx:13]: It works the same in other cases(several renders)

**Review by AwesomeFlax — APPROVED** [review]

**Review by apximax — APPROVED** [review]

**Pull request merged**